src/main/java/com/tdep/tadlab/repository/ProjectDaoImpl.java

```diff
@@ -0,0 +1,4 @@
+package com.tdep.tadlab.repository;
+
+public class ProjectDaoImpl {
```

Comment:
Some suggestions for moving files around:
- Have the `ProjectRepository`, `*Repository`, etc files go into this `/repository` directory. 
  - I don't think there's a need to have a `DAO` and a `DAOImpl` (?) Unless you're following some other template pattern -- but I think generally, you can just declare `interface ProjectRepository extends JpaRepository` (or extending whichever Spring Data JPA repository makes the most sense for you), and the queries can live directly on the interface
- I don't think you need the `/dao` directory if you end up following the above ^ 
- May want to consider making a `/dto` directory for storing your DTO classes -- these generally represent the request structure coming from the client, AKA what you're receiving from the UI. Most of the times, in the `service` layer, there's some kind of conversion from DTO to the DB model

Reply:
This also makes sense to me. Though I do wonder, is DTO basically synonymous with 'Controller' ?

Reply:
Ya, I think you could make the argument that DTO objects basically only life at the controller level (incoming requests are DTO, controller responses are also DTO)